style.scss

```diff
@@ -0,0 +1,58 @@
+$darkGreen: #21594e;
```

Comment:
Can you split this file into partials instead of putting it all in here?

Atleast, I suggest one file for the variables, another for the dino specific stuff and one for the nav